src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,26 @@
-package mate.academy.service;
-
-public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
-    public boolean login(String email, String password) {
-        return false;
-    }
-}
+package mate.academy.service;
+
+import mate.academy.model.User;
+
+public class AuthenticationService {
+    /**
+     * Imagine that some user wants to login to your site.
+     * You should check if user credentials (login and password) are valid or not.
+     * All users are stored in <code>UserService</code> class.
+     *
+     * @param email    - user's email
+     * @param password - user's password
+     * @return true if user by email exists and passed password is equal to user's password.
+     * Return false in any other cases.
+     */

```

Comment:
this part is a JavaDoc of the `login()` method which should be right above it, and constructor with the class fields should be at the beginning of the class

Reply:
Thank you, understood, I'll consider it for the next task.